formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -415,3 +417,67 @@ let asd2 = video test::false 10 [@JSX] [@foo];
 let div children => 1;
 
 ((fun () => div) ()) [] [@JSX];
+
+let myFun () =>
+  <>
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true anotherOptional=200>
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+      <Foo />
+    </Namespace.Foo>
+  </>;
+
+let myFun () => <> </>;
+
+let myFun () =>
+  <>
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true
+      anotherOptional=200
+    />
+    <Namespace.Foo
+      intended=true anotherOptional=200>
```

Comment:
Humm can this be fixed? Seeing this is a bit more satisfying:

```html
<div foo=bar1 />
<div foo=bar2 />
<div foo=bar3 />
```

Not an issue for now though.